Chap_Introduction.tex

```diff
@@ -7,162 +7,156 @@ \chapter{Introduction}
-
-% ISSUE175:  If we get rid of this, make sure these things (SMS, WLM, RM, etc) are defined early on somewhere.
+\item \deleted{the physical scale of the machines, and the corresponding number of total processes they support, is expected to reach levels approaching  1 million processes executing across 100 thousand nodes. Prior methods for initiating applications relied on exchanging communication endpoint information between the processes, either directly or in some form of hierarchical collective operation. Regardless of the specific mechanism employed, the exchange across such large applications would consume considerable time, with estimates running in excess of 5-10 minutes; and}
+\item \deleted{whether it be hybrid applications that combine OpenMP threading operations with MPI, or application-steered workflow computations, the HPC community is experiencing an unprecedented wave of new approaches for computing at exascale levels. One common thread across the proposed methods is an increasing need for orchestration between the application and the \ac{SMS} comprising the scheduler (a.k.a. the \ac{WLM}), the \ac{RM}, global file system, fabric, and other subsystems. The lack of available support for application-to-SMS integration has forced researchers to develop "virtual" environments that hide the SMS behind a customized abstraction layer, but this results in considerable duplication of effort and a lack of portability.}
 \end{itemize}
 
-% ISSUE175: Consider presenting the full goals of the PMI efforts rather than presenting (as is currently done) PMIx as an add-on to PMI-previous.
-
-% ISSUE175: Suggestion:  “PMIX is designed to support clusters of large sizes….” and then delete all text prior to this in the chapter?
-
-\ac{PMIx} represents an attempt to resolve these questions by providing an extended version of the \ac{PMI} definitions specifically designed to support clusters up to exascale and larger sizes.
-The overall objective of the project is not to branch the existing definitions -- in fact, \ac{PMIx} fully supports both of the existing PMI-1 and PMI-2 \acp{API} -- but rather to:
-
-% ISSUE175: Need to determine if PMI-1, PMI-2 API’s should be part of the standard (documented somehow?).  Or it is sufficient to say that these can easily be implemented on top of PMIx API’s?
+\deleted{ \ac{PMIx} represents an attempt to resolve these questions by providing an extended version of the \ac{PMI} definitions specifically designed to support clusters up to exascale and larger sizes.} 
+\replaced{\ac{PMIx} was intended to expand on the PMI effort,
```

Comment:
I'm not sure how accurate this statement is, but regardless I don't see a need for it in this doc. At best, it is open to debate.